configure.py

```diff
@@ -316,10 +328,13 @@ def install_local_artifacts(self, number, url):
             ]
 
         logging.debug("copying files into position")
-        for file in files:
-            from_path = os.path.join(dir_name, file[0])
-            logging.debug("copy {} to {}".format(from_path, file[1]))
-            shutil.copyfile(from_path, file[1])
+        for rel_from, rel_to in files:
```

Comment:
again, not keen on just jumping over missing files - I'd say this was an error that was worth stopping for

Reply:
Ditto - now reverted